lib/maze/helper.rb

```diff
@@ -89,18 +89,21 @@ def read_at_arg_file(argument)
       # Returns the current platform all lower-case.
       # @return Typically 'ios', 'android', 'mac' or 'browser'
       def get_current_platform
-        os = case Maze.config.farm
-             when :bs
-               Maze.config.capabilities['platformName'].downcase
-             when :sl
-               Maze.driver.capabilities['platformName']
-             else
-               Maze.config.os
-             end
-        os = os&.downcase
+        if Maze.mode == :browser
+          os = 'browser'
+        else
+          os = case Maze.config.farm
+               when :bs
+                 Maze.config.capabilities['platformName'].downcase
+               when :sl
+                 Maze.driver.capabilities['platformName']
+               else
+                 Maze.config.os
+               end
+          os = os&.downcase
```

Comment:
If we're going to use downcase anyway, can we remove it from the `:bs` option and merge `bs` and `sl`?